metrics/debug.go

```diff
@@ -19,15 +19,15 @@ var (
 	gcStats debug.GCStats
 )
 
-// Capture new values for the Go garbage collector statistics exported in
+// CaptureDebugGCStats captures new values for the Go garbage collector statistics exported in
```

Comment:
Please wrap documentation at around 80 characters. it looks very bad if you have a gigantic line longer than everything else (same below).

Reply:
Thank you - updated.